src/components/input/demo/index.tsx

```diff
@@ -330,6 +334,16 @@ export class SmoothlyInputDemo {
 				<smoothly-item value="12">December</smoothly-item>
 			</smoothly-input-select>,
 			<button onClick={async () => this.selectElement.reset()}>Reset select</button>,
+
+			<smoothly-form looks="line" onSmoothlyFormSubmit={e => console.log("Submitted", e.detail)}>
+				<smoothly-input-file placeholder="Select or drag a file here" name="input file demo">
+					{" "}
```

Comment:
last comment marked resolved without comment or doing what my comment was about. is this intended?

Reply:
I forgot to push :)